flake.nix

```diff
@@ -568,7 +568,7 @@
             cacert
             foreman
             glibcLocales
-            netcat-openbsd
+            libressl.nc
```

Comment:
What's the reason for this? `netcat-openbsd` still exists on master.
Also not too thrilled to add anything from libressl when the rest works with the well-maintained nixpkgs openssl

Reply:
netcat-openbsd did not still exist on master when I made this PR, I'm guessing that's since changed. It was an alias to libressl.nc before the alias removal.

Reply:
Looks like it [was removed in 2018, replaced with an alias](https://github.com/NixOS/nixpkgs/commits/nixos-unstable/pkgs/tools/networking/netcat-openbsd/default.nix) then re-added this may